src/components/UppyCdnExample.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import CodeBlock from '@theme/CodeBlock';
+import Admonition from '@theme/Admonition';
+
+
+const uppyVersion = '2.9.0' // todo
+
+export default function UppyCdnExample ({
+  children,
+  uppyCssName = 'uppy.min.css',
+  uppyJsName = 'uppy.min.js',
+}) {
+  let lines = [];
+
+  React.Children.toArray(children).forEach((child) => {
+    lines = [...lines, ...String(child).trim().split('\n').map((line) => line.trim())];
+  })
+
+  const linesProcessed = lines.map((line) => `  ${line}`).join('\n');
+
+  const html = `\
+<!-- 1. Add CSS to \`<head>\` -->
+<link href="https://releases.transloadit.com/uppy/v${uppyVersion}/${uppyCssName}" rel="stylesheet">
+
+<!-- 2. Add JS before the closing \`</body>\` -->
+<script src="https://releases.transloadit.com/uppy/v${uppyVersion}/${uppyJsName}"></script>
+
+<!-- 3. Initialize -->
+<div id="uppy"></div>
+
+<script>
+  var uppy = new Uppy.Core()${linesProcessed.length > 0 ? `\n\n${linesProcessed}` : ''}
```

Comment:
We shouldn't put uppy core here by default, I think it's nicer to see the complete example while writing docs and we may want to change the options for some examples.

Reply:
What should we use instead? I see all the existing examples use Uppy.Core.

Reply:
Oh you mean we should leave the script tag empty

Reply:
No I meant it feels less magical to explicitly write an example every time, so I personally wouldn't put the default uppy core in here already